src/NUnitFramework/framework/Constraints/Operators/ExactCountOperator.cs

```diff
@@ -39,17 +39,24 @@ public class ExactCountOperator : CollectionOperator
         /// <param name="expectedCount">The expected count</param>
         public ExactCountOperator(int expectedCount)
         {
+            // Collection Operators stack on everything
+            // and allow all other ops to stack on them
+            this.left_precedence = 1;
+            this.right_precedence = 10;
+
```

Comment:
I don't like the copying. Maybe the code is simple enough that' it's obvious and the copying isn't harmful, but I'm not familiar enough with precedence and the magic numbers there.

I'm actually adding this code in order *not* to change the precedence of Exactly, since `ExactCountOperator` used to inherit from `CollectionOperator` and doesn't anymore. This way the precedence of Exactly does not change pre- to post-merge.

Or rather than from `CollectionOperator`, are you asking if there's any reason to change it from the defaults of `SelfResolvingOperator`? I am not familiar enough with precedence to answer that. I assumed it should stay the same as it was when it was a `CollectionOperator` but I was hoping for your guidance on precedence.